port/unix/omrmmap.c

```diff
@@ -365,7 +492,17 @@ omrmmap_dont_need(struct OMRPortLibrary *portLibrary, const void *startAddress,
 			if (-1 == disclaim64((void *)roundedStart, roundedLength, DISCLAIM_ZEROMEM)) {
 				Trc_PRT_mmap_dont_need_disclaim64_failed((void *)roundedStart, roundedLength, errno);
 			}
-#endif /* defined(AIXPPC) */
+#elif defined(J9ZOS390)
```

Comment:
```c
#elif defined(AIXPPC) */
```

Reply:
You mean add comment `/* defined(AIXPPC) */` at this line ?

Reply:
Yes.

Reply:
Fixed.